src/credentials/GroupDataProvider.h

```diff
@@ -45,22 +45,49 @@ class GroupDataProvider
     struct GroupMapping
     {
         // The endpoint to which a GroupId is mapped.
-        EndpointId endpoint;
+        EndpointId endpoint = 0;
```

Comment:
Should not default to endpointId 0 since it's a valid value. E.G. `ClusterInfo` class uses `kInvalidEndpointId`

Reply:
Fixed.